x/finality/types/params.go

```diff
@@ -13,6 +14,7 @@ const (
 	DefaultSignedBlocksWindow = int64(100)
 	DefaultMinPubRand         = 100
 	DefaultFinalitySigTimeout = 3
+	DefaultJailDuration       = 60 * 10 * time.Second // 10 min
```

Comment:
This looks like too little to me. 25 hours or so sounds reasonable. So that jailing:

- Is a punishment in itself for being offline.
- Gives some time to the operator to properly restore service / recover liveness.

Reply:
This was copied from Cosmos SDK but yeah I agree that 1 day seems more reasonable. Fixed in [2caa4af](https://github.com/babylonlabs-io/babylon/pull/65/commits/2caa4af037b38ed3a30efbf8ae82a671ec6ddb75)